### PR TITLE
Docsp 17936 insert one type script example

### PR DESCRIPTION
**Pull request by biniona-mongodb**
## Pull Request Info
Add TypeScript Example to InsertOne Page

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-15236

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-17936-InsertOne-TypeScript-Example/usage-examples/insertOne/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### CODE REVIEW

**Review by terakilobyte — COMMENTED** [review]
For this, I'd use a different collection. The schema for movies is vastly different than what you're currently using.

**Comment by biniona-mongodb** [comment]
Thanks for the note Nathan! Updated the schema, database, and collection.

**Review by terakilobyte — APPROVED** [review]
LGTM. Thanks for the change, this makes a lot more sense for the awesome new type hinting.

**Review by jwilliams-mongo — APPROVED** [review]
@biniona-mongodb LGTM % the `movies` variables you mentioned in slack

**Comment by biniona-mongodb** [comment]
Hey @mongodb/dbx-node, could I have a review for this type script usage example?

File: source/code-snippets/usage-examples/insertOne.js [hunk 1]
```diff
@@ -10,15 +10,16 @@ async function run() {
...
-    const movies = database.collection("movies");
-    // create a document to be inserted
-    const doc = { name: "Red", town: "kanto" };
-    const result = await movies.insertOne(doc);
+    const database = client.db("insert_db");
+    const haiku = database.collection("haiku");
+    // create a document to insert
+    const doc = {
+      title: "Record of a Shriveled Datum",
+      content: "No bytes, no problem. Just insert a document, in MongoDB",
```
Comment (nbbeeken) [inline source/code-snippets/usage-examples/insertOne.js:18]: This poetry is 🔥 🔥 🔥 !

**Review by nbbeeken — APPROVED** [review]
LGTM

File: source/code-snippets/usage-examples/insertOne.js (hunk 1 above)
Comment (biniona-mongodb) [reply source/code-snippets/usage-examples/insertOne.js:18]: Lolol this is all @terakilobyte !

**Pull request merged**